### ISSUE TITLE
(feat) support to publish and record laws 

### ISSUE THREAD

**Issue by C-o-m-o-n** [Friendsofthepeople/django-recall-server#41]
## Feature
> Needs support to publish and record laws
## Task
> Create models to publish laws and their information.
> Users should be able to vote alongside their leaders

> Expected endpoints;
>> - ```law/county/create```
>> - ```law/county/update```
>> - ```law/county/mca/vote```
>>  - ```law/county/people/vote```
>> - ```law/parliament/create```
>> - ```law/parliament/update```
>> - ```law/parliament/mp/vote```
>> - ```law/parliament/senator/vote```
>> - ```law/parliament/people/vote```

## Background;
- This designer is very active and those who had promised to do recall are a bit late - i don't have a clear update yet.
- Recall also seems to have been taken by recall.ke. 

I'm thinking, what would be the situation to add support for the above features;
- Currently has the same login and admin required data needs support to publish and record laws 
- could use same module as publish mp information.
allows extensive public participation by voting alongside representative all the way and compares sentiments.
- Then uses the basis of mp trend of voting to back the call for recall. 
- using the vote to recall module for voting on bills.

In short, recalling becomes a features of a bigger project on the whole legislature at county and national level.
We'll kill two birds with one stone.

## Figma file;
- https://www.figma.com/design/l1GizjVgOwjatXXMVYXcn6/Bonga?m=auto&t=PtWr4OoxKNO3sCco-1
